README.md

```diff
@@ -1 +1,219 @@
+
+### What are the agents in this process?
+- **Issuer**: An actor who makes a claim.
+- **Holder**: An actor who receives a claim.
+- **Verifier**: An actor who verifies if the content of a claim is issued by a specific identity and held by another specific identity.
+
+### And therefore, what is a verifiable credential?
+**Credential**: Data that is needed to prove that a claim is issued by a specific identity and held by another specific identity. This data is composed of **a claim and a proof**.
+
+## Decentralized Identifiers (DIDs)
+
+A [DID](https://www.w3.org/TR/did-core/#dfn-decentralized-identifiers) is a simple text string consisting of three parts: 
+1. The `did` URI scheme identifier.
+2. The identifier for the [DID method](https://www.w3.org/TR/did-core/#dfn-did-methods).
+3. The DID method-specific identifier.
+
+![https://cdn.discordapp.com/attachments/1151100469490487307/1206967003500511343/Group_1165.png?ex=65ddee9d&is=65cb799d&hm=ada436c44d4996110fa3241c212b26dfd610a9e94c1d9015600ff9d144708c8e&](https://cdn.discordapp.com/attachments/1151100469490487307/1206967003500511343/Group_1165.png?ex=65ddee9d&is=65cb799d&hm=ada436c44d4996110fa3241c212b26dfd610a9e94c1d9015600ff9d144708c8e&)
+Currently in Bloock Identity you can use the following methods.
+- [did:polygonid](https://github.com/0xPolygonID/did-polygonid)
+- Soon they will be extended...
+
+## did:polygonid
+
+The method id called `did:polygonid` has the following characteristics.
+- Is generated from three identity trees and controlled by [Baby JubJub keys](https://docs.iden3.io/publications/pdfs/Baby-Jubjub.pdf). Therefore this key you can create with [Bloock Keys product](go_to_baby_jub_jub_keys documentation) in a managed way or by yourself locally. It will be very important to be able to control your identity.
+- Example:
+
+![https://cdn.discordapp.com/attachments/1151100469490487307/1206965230601506866/Group_1162_1.png?ex=65ddecf7&is=65cb77f7&hm=36a6fdd770f7b18bbf028cbb723b995fdcf2e95f5e2fad8f4fa4d91b4bf6ebec&](https://cdn.discordapp.com/attachments/1151100469490487307/1206965230601506866/Group_1162_1.png?ex=65ddecf7&is=65cb77f7&hm=36a6fdd770f7b18bbf028cbb723b995fdcf2e95f5e2fad8f4fa4d91b4bf6ebec&)
+Currently in Bloock Identity we have the following possibilities.
+- **did:polygonid:polygon:main**
+- Soon they will be extended...
+
+Follows the [Iden3](https://docs.iden3.io/) protocol. The communication protocol between wallets and agents is determined by [DIDComm messaging protocol](https://iden3-communication.io/).
+
+
+## Core
+These are the core components of the Bloock Identity product.
+
+**Diagrama**
+
+### The Issuer
+The issuer is an entity (person, organization, or thing) that issues Verifiable Credentials (VCs) to the Holders. VCs are cryptographically signed by the Issuer. Every VC comes from an Issuer.
```

Comment:
VCs are cryptographically signed by the Issuer -> No sempre no? Ve determinat pel did no?